src/panels/lovelace/cards/hui-entities-card.js

```diff
@@ -48,6 +51,7 @@ class HuiEntitiesCard extends EventsMixin(PolymerElement) {
     <ha-card>
       <div class='header'>
         <div class="name">[[_computeTitle(config)]]</div>
+        <hui-entities-toggle hass="[[hass]]" entities="[[config.entities]]"></hui-entities-toggle>
```

Comment:
Feature request to turn this off incoming … 😉

Reply:
what do you suggest as default?^^

Reply:
on.

Reply:
`show_header_toggle: false/true`